examples/interpreter/physl.cpp

```diff
@@ -45,6 +47,14 @@ std::string read_user_code(std::string const& path)
     return str_stream.str();
 }
 
+std::tuple<bool, std::string> get_env_physl_ir() {
+    auto e = boost::this_process::environment();
+    auto env_physl_ir = e.find("PHYSL_IR");
```

Comment:
A simple `std::getenv("PHYSL_IR")` would probably do the trick here (https://en.cppreference.com/w/cpp/utility/program/getenv) avoiding to pull in Boost.Process just for this. This would also fix the compilation errors we're seeing on CircleCI caused by missing #includes in Boost.Process.

Reply:
Right! I have a fix in the works.